requirements/runtime.txt

```diff
@@ -3,4 +3,5 @@ numpy
 Pillow
 pyyaml
 regex;sys_platform=='win32'
+setuptools>=52
```

Comment:
Which part of code relies on setuptools' version?

Reply:
In earlier versions of setuptools, the version object has no attribute named release.